operate/client/src/App/ProcessInstance/index.tsx

```diff
@@ -212,7 +215,7 @@ const ProcessInstance: React.FC = observer(() => {
           header={<ProcessInstanceHeader />}
           topPanel={<TopPanel />}
           bottomPanel={
-            <BottomPanel>
+            <BottomPanel $hasExpandedPanel={hasFlowNodeListeners}>
```

Comment:
I'm using this style prop to change the size of the bottom right panel on the CSS based on if the selected flow node has listeners or not.